utils/log_test.go

```diff
@@ -72,6 +72,19 @@ func TestLogLevelUnmarshalText(t *testing.T) {
 	})
 }
 
+func TestLogLevelkMarshalJSON(t *testing.T) {
```

Comment:
```suggestion
func TestLogLevelMarshalJSON(t *testing.T) {
```

Reply:
We can iterate over `levelStrings` in the test; that way the test will continue to work if we add/remove log levels. Same for the new `utils.Network` test.

Reply:
Done

Reply:
Make sense. Although, I don't think we need to test trivial functions. I will revert the map changes.

Reply:
Done